addon/components/column-chart/component.js

```diff
@@ -339,14 +339,14 @@ export default BaseChartComponent.extend({
         let maxValue, maxIdx, minValue, minIdx, values, nonZeroValues;
         let groups = this.get('group');
         groups.forEach((g, index) => {
-            if (this.get('showMaxMin') && _.isNumber(this.get('seriesMaxMin'))) {
```

Comment:
I'm concerned that we aren't checking whether `seriesMaxMin` is a number anymore, since later we use it as an index (see line 373). If it doesn't resolve to a number, we'll get an error there.

Reply:
that entire check is actually redundant. but yeah there are 2 spots I should be checking if it is a number

Reply:
thanks